content/_help/verify-your-identity/verify-your-identity-in-person._zh.md

```diff
@@ -85,15 +85,15 @@ Login.gov 提供到你附近件的美国邮局（USPS）亲身验证身份的选
 
 ### 如果你亲身验证身份未能成功，该怎么办？
 
-如果你未能成功地亲身验证身份，你可以再次[在网上验证你的身份，](/zh/help/verify-your-identity/how-to-verify-your-identity/) 或者联系你试图访问的机构来得到有关验证身份[具体问题的帮助。](/zh/help/specific-agencies/overview/)
+如果你无法成功亲身验证你的身份，你可以尝[试在网上或再次亲身验证你的身份](/zh/help/verify-your-identity/how-to-verify-your-identity/), 或者你可以联系你试图访问的机构，以[获取有关验证身份具体问题的帮助](/zh/help/specific-agencies/overview/)。
 
 有关亲身验证身份的小贴士：
 
 * 你的州政府颁发的身份证件或驾照绝对没有过期。我们目前不接受任何其他形式的身份证件。
 
 ### 要是亲身验证身份的截止日期已过，怎么办？
 
-如果你收到电邮，告诉你验证你身份的截止日期已过，请尝试[再次在网上验证身份。](/zh/help/verify-your-identity/how-to-verify-your-identity/)
+如果你无法成功亲身验证你的身份，你可以尝试在网上或[再次在网上验证身份。](/zh/help/verify-your-identity/how-to-verify-your-identity/)
```

Comment:
@jennyverdeyen  I am having a hard time understanding why this change was made. There are two changes to all of the other languages but I am seeing a third change for Chinese. Sorry if I am overlooking something.

Reply:
Yep this was a mistake!! Great catch, thanks!